app/assets/javascripts/utils/media_wiki_references_utils.js

```diff
@@ -0,0 +1,79 @@
+import { ORESSupportedWiki } from './article_finder_language_mappings';
+import { queryUrl } from './article_finder_utils';
+import { chunk } from 'lodash-es';
+import { getReferencesCount } from './revision_utils';
+
+// given a particular wiki, find all the references added for each revision in it
+const fetchReferencesAddedFromWiki = async (wiki_url, revisions) => {
+  // eslint-disable-next-line no-console
+  console.log(`Fetching references information from ${wiki_url}`);
+
+  const list = wiki_url.split('.');
+  let wiki;
+  if (list.length === 3) {
+    wiki = { language: list[0], project: list[1] };
+  } else {
+    wiki = { project: list[0] };
+  }
+  if (
+    !ORESSupportedWiki.projects.includes(wiki.project)
+  && !ORESSupportedWiki.languages.includes(wiki.language)
+  ) {
+    // wiki does not support ORES
+    return;
+  }
+  let suffix;
+  if (wiki.project === 'wikidata') {
+    suffix = 'wikidatawiki';
+  } else {
+    suffix = `${wiki.language}wiki`;
+  }
+  const API_URL = `http://ores.wikimedia.org/v3/scores/${suffix}`;
+  const revids = revisions.filter(revision => revision.ns === 0).map(revision => `${revision.parentid}|${revision.revid}`);
+  const chunks = chunk(revids, 25);
+
+
+  const promises = [];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const revid_chunk of chunks) {
+    const params = {
+      revids: revid_chunk.join('|'),
+      features: true,
+      models: 'articlequality'
```

Comment:
For wikidata, the model is called `itemquality` instead. Querying for `articlequality` results in a 404.